src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```
@@ -196,8 +196,23 @@ public Path getJavaAppEngineSdkPath() {
     return getSdkPath().resolve(JAVA_APPENGINE_SDK_PATH);
   }
 
-  private Path getWindowsPythonPath() {
-    return getSdkPath().resolve(WINDOWS_BUNDLED_PYTHON);
+  // https://github.com/GoogleCloudPlatform/appengine-plugins-core/issues/189
+  @VisibleForTesting
+  Path getWindowsPythonPath() {
+    String cloudSdkPython = System.getenv("CLOUDSDK_PYTHON");
+    if (cloudSdkPython != null) {
+      Path cloudSdkPythonPath = Paths.get(cloudSdkPython);
+      if (Files.isExecutable(cloudSdkPythonPath)) {
+        return cloudSdkPythonPath;
+      }
+    }
+    
+    Path pythonPath = getSdkPath().resolve(WINDOWS_BUNDLED_PYTHON);
+    if (Files.isExecutable(pythonPath)) {
```

ditto... if it's not executable, something is really wrong with the installation of the Cloud SDK. Let it crash.